.clang-format

```diff
@@ -15,5 +15,4 @@ AllowShortLoopsOnASingleLine: false
 BinPackParameters: false
 AlwaysBreakTemplateDeclarations: Yes
 ReflowComments: false
-BinPackArguments: false
-BinPackParameters: false
+BinPackArguments: false
```

Comment:
The `BinPackArguments` and `BinPackParameters` are two different options (see https://clang.llvm.org/docs/ClangFormatStyleOptions.html#binpackarguments), please undo this change.

Reply:
I am aware, please see line 15 and 19 - doubled BinPackParameters.

Reply:
Ah, right, thanks. Could you then do this change in a separate PR?

Reply:
Sure, I'll put it in a separate PR :)